GUI/QtQuick/FeeSuggModel.cpp

```diff
@@ -39,6 +39,17 @@ int FeeSuggestionModel::columnCount(const QModelIndex &) const
 
 QVariant FeeSuggestionModel::data(const QModelIndex& index, int role) const
 {
+   if (index.row() == data_.size()) {
+      switch (role) {
+      case TextRole:
+         return tr("Manual Fee Selection");
+      case BlocksRole:
+      case TimeRole:
+      case ValueRole:
+         return QString::fromLatin1("1.0 s/b");
```

Comment:
This should be also translatable. Also, why is it fixed to value 1.0?
Can we have "s/b" as a static unselectable and un-editable text drawn in the box background?